lib/rewards/server/entities/rewards-entity.js

```diff
@@ -0,0 +1,57 @@
+/**
+ *
+ * Reldens - RewardsEntity
+ *
+ */
+
+const {EntityProperties} = require('../../../game/server/entity-properties');
```

Comment:
cs: use a space between the braces

Reply:
done

Reply:
`const {EntityProperties}` > `const { EntityProperties } `

Reply:
make sure to change the style in all the files